<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAboutDeletingStation</class>
 <widget class="QDialog" name="DialogAboutDeletingStation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Удаление станции</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Название станции</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Номер маршрута</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
